cypress/integration/table.cy.js

```diff
@@ -153,77 +206,43 @@ const assertDimensions = (inputDimensions) => {
     }
 }
 
-describe('table', () => {
-    beforeEach(() => {
-        cy.visit('/', EXTENDED_TIMEOUT)
-
-        // remove org unit
-        cy.getBySel('layout-chip-ou').findBySel('dimension-menu-button').click()
-        cy.containsExact('Remove').click()
-    })
-    it('click on column header opens the dimension dialog', () => {
-        const dimensionName = TEST_DIM_TEXT
-
-        selectEventProgramDimensions({
-            ...event,
-            dimensions: [dimensionName],
-        })
-
-        const testDimensions = mainAndTimeDimensions.map(
-            (dimension) => dimension.label
-        )
+const init = () => {
+    cy.visit('/', EXTENDED_TIMEOUT)
 
-        // add main and time dimensions
-        testDimensions.forEach((label) => {
-            cy.getBySel('main-sidebar')
-                .contains(label)
-                .closest(`[data-test*="dimension-item"]`)
-                .findBySel('dimension-menu-button')
-                .invoke('attr', 'style', 'visibility: initial')
-                .click()
+    // remove org unit
+    cy.getBySel('layout-chip-ou').findBySel('dimension-menu-button').click()
+    cy.containsExact('Remove').click()
+}
 
-            cy.contains('Add to Columns').click()
+describe(['<40'], 'table', () => {
+    beforeEach(init)
+    it('click on column header opens the dimension dialog', () => {
+        programDimensions.push({
```

Comment:
I would say that we just don't test this since the value is a bug.

Reply:
Also, the test is failing:  
![image](https://user-images.githubusercontent.com/6113918/199267181-b3981d8e-289b-4022-a5a9-ea1b0d6bc784.png)